src/GovUk.Education.ExploreEducationStatistics.Common.Tests/Utils/GeographicLevelUtilTests.cs

```diff
@@ -0,0 +1,67 @@
+#nullable enable
+using System.Collections.Generic;
+using System.Linq;
+using GovUk.Education.ExploreEducationStatistics.Common.Model.Data;
+using GovUk.Education.ExploreEducationStatistics.Common.Services;
+using GovUk.Education.ExploreEducationStatistics.Common.Utils;
+using Xunit;
+
+namespace GovUk.Education.ExploreEducationStatistics.Common.Tests.Utils;
+
+public static class GeographicLevelUtilTests
+{
+    public class CsvColumnTests
+    {
+        [Fact]
+        public void CsvColumns()
+        {
+            var columns = GeographicLevel.Country.CsvColumns();
+
+            Assert.Equal("country_code", columns.Code);
+            Assert.Equal("country_name", columns.Name);
+        }
+
+        [Fact]
+        public void CsvNameColumn()
+        {
+            Assert.Equal("country_name", GeographicLevel.Country.CsvNameColumn());
+        }
+
+        [Fact]
+        public void CsvCodeColumn()
+        {
+            Assert.Equal("country_code", GeographicLevel.Country.CsvCodeColumn());
+        }
+
+        [Fact]
+        public void CsvOtherColumns()
+        {
+            var columns = GeographicLevel.LocalAuthority.CsvOtherColumns();
+
+            Assert.Single(columns);
+            Assert.Equal("old_la_code", columns[0]);
```

Comment:
Do we want to check that there isn't any other items in this list?

Reply:
It already has as we've got an `Assert.Single` above :wink:

Reply:
Woops, missed that one!